package.json

```
@@ -35,6 +35,7 @@
 		"prebuild": "yarn clean",
 		"build": "lerna run build --stream",
 		"build:web": "lerna run build:web --stream",
+		"build:docs": "cd docs && yarn install && yarn build",
```

```suggestion
		"build:docs": "yarn --cwd docs install && yarn build",
```

I'm not sure if this works on windows

---

couldnt get this to work on my mac book, it just installed and built the package in root, instead of the docs. think i ran into this issue?

---

Unfortunately, it seems that the option `--cwd` is buggy. Here are some people had reported it not working inside package.json and some others said it not working at all https://github.com/yarnpkg/berry/issues/2346. On my Linux Mint, it did not work in the command line.

---

The main issue with using `cd` is it's not platform agnostic, however, being that `--cwd` is not reliable and this command is mostly going to be ran by CI, this seems okay in lieu of a better option

---

I think `cd` is fine to use. As far as I experienced it works on Linux, Mac and with latest windows Powershell as well.